modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js

```diff
@@ -55,8 +55,12 @@ AUI.add(
 					initializer: function() {
 						var instance = this;
 
+						var evaluator = instance.get('evaluator');
+
 						instance._eventHandlers.push(
+							evaluator.after('evaluationEnded', A.bind('_loadValidationFieldType', instance)),
```

Comment:
Hey @alinedoleron , is really need listen the `evaluationEnded` event?

 Thx

Reply:
Hi @rafaprax. I investigated and we should keep listening to this event because it is fired when changing type from Integer to Double.

Reply:
@alinedoleron Sorry. I forgot that. Thanks for checking.